app/controllers/admin/api_controller.rb

```diff
@@ -0,0 +1,61 @@
+class Admin::ApiController < ApplicationController
+  before_filter :admin_only
+  before_filter :check_for_cancel, only: [:create, :update]
+
+  def index
+    @api_keys = if params[:query]
+                  sql_query = "%" + params[:query] + "%"
+                  ApiKey.where("name LIKE ?", sql_query)
+                        .order("name")
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.